packages/cli/src/ceramic-daemon.ts

```diff
@@ -273,39 +276,39 @@ class CeramicDaemon {
   }
 
   /**
-   * Get all document records
+   * Get all document commits
    */
-  async records (req: Request, res: Response, next: NextFunction): Promise<void> {
+  async commits (req: Request, res: Response, next: NextFunction): Promise<void> {
     const docId = DocID.fromString(req.params.docid)
     try {
-      const records = await this.ceramic.loadDocumentRecords(docId)
-      const serializedRecords = records.map((r: any) => {
+      const commits = await this.ceramic.loadDocumentCommits(docId)
+      const serializedCommits = commits.map((r: any) => {
         return {
           cid: r.cid,
-          value: DoctypeUtils.serializeRecord(r.value)
+          value: DoctypeUtils.serializeCommit(r.value)
         }
       })
 
-      res.json({ docId: docId.toString(), records: serializedRecords })
+      res.json({ docId: docId.toString(), commits: serializedCommits })
     } catch (e) {
       return next(e)
     }
     next()
   }
 
   /**
-   * Apply one record to the existing document
+   * Apply one commit to the existing document
    */
-  async applyRecord (req: Request, res: Response, next: NextFunction): Promise<void> {
-    const { docId, record, docOpts } = req.body
-    if (!docId && !record) {
-      res.json({ error: 'docId and record are required in order to apply record' })
+  async applyCommit (req: Request, res: Response, next: NextFunction): Promise<void> {
+    const { docId, commit, docOpts } = req.body
+    if (!docId && !commit) {
```

Comment:
huh, this should probably be `||` not `&&`?  Not related to this change though

Reply:
good catch though. let me see if it survives the tests